### ISSUE TITLE
These are very outdated and there is no real complete example anywhere that we can find

### ISSUE THREAD

**Issue by genyded** [feathersjs/feathers-demos#24]
We cannot find a solid and complete example anywhere to use as a Feathers reference app. The docs are OK and getting better with with auk reorg, but they:

1. Only show the simplest examples usually in app.js in isloated scenarios
2. Lack completeness even in those simple examples (e.g. talk mostly about what `can` be done and not so much `how`
3. Do not show complete example files (with requires and such). so when you see something like a reference you have no idea where it even came from

It would be great to have a simple app that has auth, 3 or 4 services that interact with each others, maybe a non-db service in the mix that sends email, talks to an external api, or something similar. It should be laid out in the full preferred app structure. So some of the services could have hooks that talk to other services, modify queries and such, and all the pieces are in there owns files and referenced like a real app.

It seems time spent here with the focus on the Feathers parts and maybe less on the other integration pieces like clients, comparisons to other frameworks, and such would be optimal. Show us the full way(s) to glue all the core feathers pieces together in a single semi-real-world app. Once we get a handle on that and most of us can handle the client stuff in our flavor of choice, continue to know Feathers JS blows the doors off of Sails or whoever, and do the integrations. If getting what we need to (fully) integrate with on the Feathers side that seems to be missing in even a basic form.

It doesn't have to be anything `huge`, just enough to show all the concepts actually working in tandem in a sort of `real` app with a few different services talking to each other, some basic auth, and so on. Not one Todo service or Voting service displaying real-time updates in some client. We'd be happy to help with this as much as we can, but are time constrained trying to figure all this out for ourselves and improve our rethink and Vue knowledge in parallel. We also don't really `know` all of Feathers stuff this well enough (yet) to tackle something like this ourselves.

Not even sure if this is the right place for this issue since most of the stuff here hasn't been touched in months, but I guess we'll start here and see. If we're the only ones wanting/needing something like this, maybe we're missing something and if so we'll shut up, but we won't go away because we love flying ;-)

**Comment by marshallswain** [comment]
@genyded some of what you're asking for is already in the works.  I've been working on a seamless Feathers + VueJS experience for the last few weeks.  It's really coming quite nicely.  I'll keep you posted.

**Comment by ekryski** [comment]
@genyded some great feedback! Thanks! I wrote most of these demo examples in responses to people's feedback but didn't get much feedback on them. 1 single example that shows a more realistic app is going to be easier to keep up to date.

I think this is the route we're are going to take with the `feathers-chat` example. Have multiple stages, so you can see how an app progresses.

**Comment by pedrogk** [comment]
I completely agree with @genyded. I am trying to get feathers auk to use Auth0 and everybody points me to the example here but I can't sort it out because as mentioned, it isn't really complete/well structured, also I don't know if it is still valid with auk. Hopefully updated examples for auth are coming. Let me know if I can help somehow.

**Comment by paul42** [comment]
@pedrogk agree with you entirely - when I run the code in the example, it works, but when I try to port to my new app (using the same version of feathers-authenticate and passport-auth0), made with feathers generate, I get a 404 when It tries to travel to /auth/auth0 - yet for the life of me I can't find any mention of where or how it sets up that auth route.
